docs/sink-configuration.md

```diff
@@ -124,3 +124,12 @@ echo '["gcm", "influxdb:http://monitoring-influxdb:8086"]' | curl \
     -H "Accept: application/json" -H "Content-Type: application/json" \
     https://<master-ip>/api/v1/proxy/namespaces/kube-system/services/monitoring-heapster/api/v1/sinks
 ```
+
+## Using multiple sinks
+
+It can activate multiple sinks described in the above sections for a heapster by using multiple `--sink=...` flags.
```

Comment:
How about:

    Heapster can be configured to send data to multiple sinks by specifying the`--sink=...` flag multiple times.